core/rawdb/database.go

```diff
@@ -298,6 +367,32 @@ func (s *stat) Size() string {
 func (s *stat) Count() string {
 	return s.count.String()
 }
+func AncientInspect(db ethdb.Database) error {
```

Comment:
seems this function never return an error

Reply:
Done